lib/contentful.js

```diff
@@ -208,31 +208,39 @@ module.exports.renderMessageForPhoenixCampaign = function (phoenixCampaign, msgT
  * @param  {type} campaignId
  * @return {Promise}
  */
-module.exports.getAllFieldsForCampaign = function getAllFieldsForCampaign(campaignId) {
+module.exports.getAllTemplatesForCampaignId = function getAllTemplatesForCampaignId(campaignId) {
   return new Promise((resolve, reject) => {
     const campaignFieldNames = Object.keys(config.campaignFields);
     const campaignContent = {};
-    const campaignFields = {};
+    const templates = {};
     return exports.fetchCampaign(campaignId)
       .then((contentfulCampaign) => {
         campaignContent.overrides = contentfulCampaign.fields;
+        // TODO: Pass the default Campaign, to avoid fetching every time we call this.
+        // GET campaigns could fetch default once, instead of every time we get each Campaign.
         return exports.fetchCampaign(defaultCampaignId);
       })
       .then((defaultContentfulCampaign) => {
         campaignContent.defaults = defaultContentfulCampaign.fields;
         campaignFieldNames.forEach((key) => {
           const fieldName = config.campaignFields[key];
-          campaignFields[fieldName] = {};
+          // This is a hack to avoid migrating our Contentful data in order to rename our fields.
+          // We want to return template values as 'askQuantity', not 'askQuantityMessage'.
+          const templateName = fieldName.split('Message')[0];
```

Comment:
I'm going to move this into a `helpers` function and add tests. We should also return the same value in our in the `reply.template` response of `POST /receive-message`:
```
{
  "success": {
    "code": 200,
    "message": "@dev Got it! Now text back a caption for your photo (think Instagram)! Keep it short & sweet, under 60 characters please.",
    "template": "askCaptionMessage"
  }
}
```

Reply:
I'll add these in a new PR